src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -683,14 +689,16 @@ static int ask_on_consoles(char *argv[]) {
         }
 
         /* Wait for an agent to exit. */
+        siginfo_t status = {};
```

Comment:
Maybe, better to move the declaration in the `for` loop below, by also moving the check `is_clean_exit()` in the loop before `break`.